services/addons/images/count_metric/sample.env

```diff
@@ -1,4 +1,31 @@
-LOGGING_LEVEL = "INFO"
+LOGGING_LEVEL = 'INFO'
+
+ENABLE_EMAILER = 'True'
+
+# If ENABLE_EMAILER is 'True', set the following environment variables
+
+DEPLOYMENT_TITLE = 'JPO-ODE'
+
+# POSTGRES DATABASE VARIABLES
+PG_DB_HOST = ':5432'
```

Comment:
Looks like these postgres variables are duplicated below - would we be able to pull out any shared variables to the top of sample.env and only have unique variables under each ENABLE_EMAILER section?

Reply:
this pr relies on the emailer so this branch will be updated when it gets merged.

Reply:
I see what you mean by reducing redundancies but I thought this was an acceptable redundancy just to make it clear which feature uses what. We will be removing the counter in the future and I wanted to make it easier to just delete it all without having to think on what needs to be refactored.